src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -476,50 +507,133 @@ public void testUpdateDistinctAttributeSettings() throws Exception {
     @Test
     @DisplayName("Test reset distinct attribute settings")
     public void testResetDistinctAttributeSettings() throws Exception {
-        Index index = createIndex("testUpdateDistinctAttributeSettings");
+        Index index = createIndex("testResetDistinctAttributeSettings");
         String initialDistinctAttribute = index.getDistinctAttributeSettings();
         String newDistinctAttribute = "title";
 
         index.waitForTask(index.updateDistinctAttributeSettings(newDistinctAttribute).getUid());
         String updatedDistinctAttribute = index.getDistinctAttributeSettings();
 
-        assertEquals(newDistinctAttribute, updatedDistinctAttribute);
-        assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
-
         index.waitForTask(index.resetDistinctAttributeSettings().getUid());
         String distinctAttributeAfterReset = index.getDistinctAttributeSettings();
 
+        assertEquals(newDistinctAttribute, updatedDistinctAttribute);
+        assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
+
         assertNotEquals(updatedDistinctAttribute, distinctAttributeAfterReset);
         assertNotEquals(initialDistinctAttribute, updatedDistinctAttribute);
     }
 
+    /** Tests of the typo tolerance setting methods */
     @Test
-    @DisplayName("Test reset ranking rules when null value is passed")
-    public void testUpdateRankingRuleSettingsUsingNull() throws Exception {
-        Index index = createIndex("testUpdateRankingRuleSettingsUsingNull");
-        String[] initialRankingRule = index.getRankingRuleSettings();
-        String[] newRankingRules = {
-            "typo",
-            "words",
-            "sort",
-            "proximity",
-            "attribute",
-            "exactness",
-            "release_date:desc",
-            "rank:desc"
-        };
+    @DisplayName("Test get typo tolerance settings by uid")
+    public void testGetTypoTolerance() throws Exception {
+        Index index = createIndex("testGetTypoTolerance");
+        Settings initialSettings = index.getSettings();
+        TypoTolerance initialTypoTolerance = index.getTypoToleranceSettings();
 
-        index.waitForTask(index.updateRankingRuleSettings(newRankingRules).getUid());
-        String[] newRankingRule = index.getRankingRuleSettings();
+        assertEquals(
+                initialSettings.getTypoTolerance().getDisableOnWords().length,
```

Comment:
Why not just assert by the value? It does not work 😬 ?

Reply:
I can't assert just `getDisableOnWords()` because is a `String[]` I could assert every value in the array but I felt it was too much than just comparing the length of the array don't you think?